nixos/modules/virtualisation/incus.nix

```
@@ -335,6 +335,24 @@ in
           include ${cfg.lxcPackage}/etc/apparmor.d/lxc-containers
         '';
       };
+      includes."abstractions/base" = ''
```

```suggestion
      includes."abstractions/incus" = ''
```

base seems generic, was that chosen for a reason?